terraformprovider/resource_object_group.go

```diff
@@ -110,6 +110,158 @@ func resourceObjectGroup() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 							ForceNew: true,
+							Description: "Column selection type: whitelist, blacklist, regex.",
+						},
+						"includes": {
```

Comment:
- [ ] Make "includes" and "patterns" mutually exclusive. The API allows both filters to be present at once, but it uses the AND logic, including only fields that satisfy both, which will be confusing to use.

Reply:
Could you check how API reacts if you set both? Because if we can return a meaningful API error - that might be sufficient

Reply:
From talking with chaossearch, it is allowed to set both filters at the same time. I want to explicitly prohibit it, because the filters are combined with the AND logic - a field will show up in the view only if it's present in both filters. This makes it confusing and error prone imo.

Reply:
There's a way to validate this using customDiff: https://github.com/hashicorp/terraform-plugin-sdk/issues/233

Reply:
If I remember correctly, Terraform recommends following the API logic as close as possible, so I would not bother with this validation if API allows it. Instead this quirk can be just mentioned in the description of the fields